service/controller/v25/resource/tccp/main_stack.go

```diff
@@ -1,71 +0,0 @@
-package tccp
-
-import (
-	"context"
-
-	"github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
-	"github.com/giantswarm/microerror"
-
-	"github.com/giantswarm/aws-operator/service/controller/v25/adapter"
-	"github.com/giantswarm/aws-operator/service/controller/v25/controllercontext"
-	"github.com/giantswarm/aws-operator/service/controller/v25/key"
-	"github.com/giantswarm/aws-operator/service/controller/v25/templates"
-)
-
-func (r *Resource) getMainGuestTemplateBody(ctx context.Context, customObject v1alpha1.AWSConfig, stackState StackState) (string, error) {
```

Comment:
There is nothing like a main template anymore. There is only the stack template now.